parquet-column/src/main/java/org/apache/parquet/column/statistics/LongStatistics.java

```diff
@@ -97,6 +109,19 @@ public Long genericGetMax() {
     return max;
   }
 
+  @Override
+  public LongComparator comparator() {
+    return LongComparators.NATURAL_COMPARATOR;
```

Comment:
No need to override here, the type can provide this.

Reply:
Good catch, I've missed to remove it after rewriting to use my PrimitiveComparator class.